### ISSUE TITLE
Create schema.md to hold schema documentation

### ISSUE THREAD

**Issue by digitalman2112** [nationofmakers/maker-event-playbook#16]
I didn't want to start adding files until we get the branch cleaned out...

**Comment by digitalman2112** [comment]
Current working schema file: https://docs.google.com/document/d/14IOSMaYdNdrGj_C7gZKrWsi19X_8s7E3KyJkOrhsz98/edit?usp=sharing

**Comment by mimkinetic** [comment]
Ian,
Do you see event documentation (video, photos, earned media from press)
fitting into one of the current General Event Topics?  Expanding reach and
dwell time in the public eye seems often overlooked by new event planners.

Have a great day and keep pedaling,

Joseph

https://www.linkedin.com/in/josephfloyd


-------         o

------ _ `\ <, _

----- ( • ) / ( • )

 --------------------

"Those who wish to control their own lives

and move beyond existence as mere

clients and consumers - those people

ride a bike."  ~ Wolfgang Sachs

**Comment by digitalman2112** [comment]
Yes, I would expect that to be part of the schema, I'll make a placeholder for now
